kuma/landing/templates/landing/promote_buttons.html

```diff
@@ -284,7 +284,7 @@
         // Update the button code.
         $('#button-code').val([
             '<a href="', sel.tracked_url, '" title="', sel.alt_text, '">',
-            '<img src="{{ request.build_absolute_uri(MEDIA_URL) }}img/promote/promobutton_', sel.id, '.png" ', 
+            '<img src="{{ request.build_absolute_uri(STATIC_URL) }}img/promote/promobutton_', sel.id, '.png" ',
```

Comment:
I think we're fine with this now, but we should open a follow-up ticket to remove all uses of `STATIC_URL` to be able to switch out the staticfiles storage backend when we move to Cloudfront or some other form of backend that generates URLs to files programmatically via an API.